underdocs-renderer/src/main/resources/main.css

```diff
@@ -26,16 +27,57 @@
     --member-type-color: #008000;
 
     --parameter-type-color: #008000;
+    
+    --token-string-color: #A31515;
+    --token-keyword-color: #0000FF;
+    --token-symbol-color: #36ACAA;
+    --token-deleted-color: #9A050F;
+    --token-class-name-color: #2B91AF;
+
+    --remote-link-svg-filter: none;
+
+    --transition-time: 1s;
 
     --main-width: 80ch;
 }
 
-html {
+body {
     font-size: 1.1rem;
     line-height: 1.5;
 
     color: var(--text-color);
     background-color: var(--background-color);
+
+    transition: var(--transition-time);
+}
+
+body.night {
+    --background-color: #1E1E1E;
+
+    --text-color: #E5E5E5;
```

Comment:
This color is much more pleasing for the eye than the previous one. Good choice!